pacman-nodejs-app/tools/migrate/pacman.sh

```diff
@@ -2,15 +2,33 @@
 set -e
 
 function add_new_mongo_instance {
-    MONGO_SRC_POD=$(kubectl --context ${SRC_CONTEXT} get pod \
-    --selector="name=mongo" --output=jsonpath='{.items..metadata.name}')
+    
+	MONGO_SRC_POD=$(kubectl --context ${SRC_CONTEXT} get pod \
+		--selector="name=mongo" \
+		--output=jsonpath='{.items..metadata.name}')
+	
+	MONGO_DST_POD=$(kubectl --context ${DST_CONTEXT} get pod \
+        	--selector="name=mongo" \
+		--output=jsonpath='{.items..metadata.name}')
 
-    MONGO_DST_POD=$(kubectl --context ${DST_CONTEXT} get pod \
-        --selector="name=mongo" \
-        --output=jsonpath='{.items..metadata.name}')
+	if valid_ip ${PACMAN_SRC_PUBLIC_ADDRESS}; then
```

Comment:
I think you only want to check the destination address. That is, if the destination address is not a `valid_ip` then you'll assume it's a DNS so you'll need to perform a DNS resolution check before adding it to the mongo replica set. That way it will also enable you to migrate from AWS to AWS.